ruby/rails/weak_custom_key.yml

```
@@ -0,0 +1,31 @@
+patterns:
+  - pattern: |
+      encrypts $<...> key: $<LENGTH>
```

I think this is equivalent:
```suggestion
      encrypts key: $<LENGTH>
```